### ISSUE TITLE
Use oss-fuzz

### ISSUE THREAD

**Issue by marshallpierce** [alicemaz/rust-base64#98]
They'll accept rust-base64: https://github.com/google/oss-fuzz/issues/2145

**Comment by denniskempin** [comment]
I am looking through the supply chain of some of my projects to look for ways to contribute back. 

Are you still interested in this? I just onboarded a project to oss-fuzz, I'd be happy to do another one.

**Comment by marshallpierce** [comment]
@denniskempin Yes, that would be great, thanks! This will be especially handy once SIMD-based (and therefore unsafe) engines are a thing.

**Comment by denniskempin** [comment]
Hi @marshallpierce! Onboarding was pretty easy, I got a working commit here: https://github.com/denniskempin/oss-fuzz/commit/8ccaa5fe3cf22fb35a3911883d865f9b91eaa827

In order to interact with oss-fuzz infrastructure and receive access to the filed bugs you will need to provide a google account. See: https://google.github.io/oss-fuzz/faq/#why-do-you-require-a-google-account-for-authentication

Could you email me with a google email address you would like to use and a list of additional emails you'd like to add to the CC? My email is denniskempin@google.com

**Comment by denniskempin** [comment]
Pull request is up. Sorry for the delay: https://github.com/google/oss-fuzz/pull/10693

**Comment by denniskempin** [comment]
It sounds like they would prefer to use https://google.github.io/clusterfuzzlite/ for base64 (which makes sense, a centralized infra for all OSS fuzzing is hard to maintain). 

Unfortunately it does not support CircleCI. This could be set up with GitHub actions on this repo.

It may not be too much work to implement CircleCI support into ClusterFuzzLite though. That may be generally useful beyond rust-base64 anyway.

**Comment by marshallpierce** [comment]
If it's not too bad to use GitHub Actions just for fuzzing, that sounds like a good compromise -- I like fuzzing more than I dislike GHA. :) If that's not workable, then I'm fine with switching the whole setup to GHA.
